### PR TITLE
Sucurity Config

### PR DESCRIPTION
**Pull request by xdudasr**
Sucurity Config

### CODE REVIEW

File: web/pom.xml [hunk 1]
```diff
@@ -60,6 +60,38 @@
...
+            <artifactId>spring-security-config</artifactId>
+            <version>5.1.2.RELEASE</version>
+        </dependency>
+        <!-- https://mvnrepository.com/artifact/org.springframework.security/spring-security-core -->
+        <dependency>
+        <groupId>org.springframework.security</groupId>
+        <artifactId>spring-security-core</artifactId>
+        <version>2.0.2</version>
+        </dependency>
+        <dependency>
```
Comment (tyrylu) [inline web/pom.xml:64]: Proč ta samá dependency 2x se stejnou verzí?

**Review by tyrylu — CHANGES_REQUESTED** [review]
Skoro bych se bál, že ty dvě verze stejný dependency můžou dělat bordel, minimálně jsou matoucí.

**Comment by rutor** [comment]
LGTM pull request analysis was skipped for f63182f71e3040eee206a73239e45fd9def2333b by [rutor](https://github.com/rutor). Analysis of future commits will happen as normal.

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by rutor** [comment]
LGTM pull request analysis was skipped for 596008f499fb18090239f128f4ef8a65be4b5ce5 by [rutor](https://github.com/rutor). Analysis of future commits will happen as normal.

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Review by tyrylu — APPROVED** [review]

**Comment by rutor** [comment]
This pull request **introduces 1 alert** when merging 596008f499fb18090239f128f4ef8a65be4b5ce5 into d831be182e8f74a152ced9a1779ead36ecd36dfb - [view on LGTM.com](https://lgtm.com/projects/g/rutor/PA165-projekt/rev/pr-b2d58f1b3a228c29cd67111ce65e1d742cd8a8c0)

**new alerts:**

* 1 for URL redirection from remote source

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**